README.md

```diff
@@ -1,98 +1,170 @@
+        - [Signing in as a user with specific roles](#signing-in-as-a-user-with-specific-roles)
+    - [Prometheus Integration](#prometheus-integration)
+        - [Install Prometheus](#install-prometheus)
+            - [Prerequisites](#prerequisites-1)
+            - [Installation](#installation-1)
+            - [Usage](#usage)
+            - [Enabling Services](#enabling-services)
+
+<!-- markdown-toc end -->
+
+
+# OPAL-HELM
+
+This repo contains all the deployment helm charts and configuration files for deploying the Open Platform for Avionics Learning or OPAL.
+
 ## Getting Started with Opal-helm Deployment
 
-### ArgoCD
-ArgoCD is a Kubernetes controller, responsible for continuously monitoring all running applications and comparing their live state to the desired state specified in the Git repository.
+### Prerequisites
 
-The opal team uses Argo CD to make sure our repository is fully managed and monitored for the most updated changes and features.
+- A Kubernetes Cluster (or Minikube if deploying locally)
+- Kubectl
+    - https://kubernetes.io/docs/tasks/tools/#kubectl
+- K9s (optional, but highly recommended)
+    - https://k9scli.io/topics/install/
+- Helm
+    - https://helm.sh/docs/intro/install/
+- a valid authentication token for `registry1.dso.mil`
+- Clone https://github.com/309theddge/opal-helm
 
-## Deploying ArgoCD 
+#### Additional Prerequisites for Local Deployment
 
-The following packages are required before proceeding 
- - minikube 
+- Minikube
     - https://minikube.sigs.k8s.io/docs/start/
-- docker 
+- Docker (DO NOT install Docker Desktop if using Linux)
     - https://docs.docker.com/engine/install/
-- kubectl 
-    - https://kubernetes.io/docs/tasks/tools/
-- helm 
-    - https://helm.sh/docs/intro/install/
+- Clone https://github.com/309theddge/k8s-utils
+- Clone https://github.com/309theddge/argo
+
+#### Optional Extras
+- Prometheus Stack
+    - https://github.com/309theddge/kube-prometheus
+
+### Local Installation
 
-- You may need more packages depending on your current environment setup
+#### Starting Minikube
 
-### Starting minikube
-From a terminal with administrator access run the following command:
->$minikube start
+Run the following command in your terminal:
+
+``` bash
+minikube start
+```
 
 If minikube fails to start, use the minikube documentation to troubleshoot any issues 
 https://minikube.sigs.k8s.io/docs/start/
 
-### Setting up ArgoCD regcred
- change directory to the regcred-init directory and run the following command:
+#### ArgoCD
+
+ArgoCD is a continuous deployment (CD) tool used to manage and provide automatic updates for Kubernetes applications
+
+The opal team uses Argo CD to ensure OPAL deployments are always up-to-date
+
 
->$helm install argoregcred . --create-namespace -n argocd
+##### Setting up ArgoCD regcred
+
+change directory to the regcred-init directory and run the following command:
+
+``` bash
+helm install argo-regcred . --create-namespace -n argocd
+```
+
```

Comment:
Where is the `regcred-init` directory?

Reply:
Seems to be in a different git repo, but is not mentioned

Reply:
Part of k8s-utils, mentioned in prereqs